common/docker-compose.yml

```diff
@@ -15,6 +15,7 @@ elasticsearch:
     - "http.host=0.0.0.0"
     - "transport.host=0.0.0.0"
     - "cluster.name=wellcome"
+    - "ES_JAVA_OPTS=-Xms750m -Xmx750m"
```

Comment:
This doesn't need to be here, it stops the common tests failing so often for me.